app/components/CardBet.tsx

```diff
@@ -27,6 +27,12 @@ interface CardBetProps extends PropsWithChildren {
   userBets: UserBets;
 }
 
+export const tagSchemeColors: Record<number, TagProps['colorScheme']> = {
+  0: 'success',
+  1: 'danger',
+  [Market.INVALID_ANSWER]: 'quaternary',
+};
```

Comment:
Is it worth transforming this into a utils fn? 
We use a similar logic at:
`app/(main)/markets/MarketActivity.tsx`

The piece of code I'm talking about:
```tsx
const TAG_COLOR_SCHEMES: { 0: TagColorSchemeProp; 1: TagColorSchemeProp } = {
  0: 'success',
  1: 'danger',
};
```

Reply:
I think an object is ok for this. Where should we declare it?